src/components/pages/helper/googlemap.js

```diff
@@ -1,27 +1,108 @@
-import React, { Component } from 'react';
-import { Map, GoogleApiWrapper } from 'google-maps-react';
+import React, {Component } from 'react';
+import { Map, GoogleApiWrapper, Marker } from 'google-maps-react';
+import Pagination from "react-bootstrap/Pagination";
 
 const mapStyles = {
-  width: '100%',
-  height: '100%'
+  width: '100vw',
+  height: '60vh'
 };
 
 export class MapContainer extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            error: null,
+            isLoaded: false,
+            items: [
+                {
+                    "items": [
+                        {
+                            "name": "Freddy Fuego Hausmannsgate",
+                            "category": "food",
+                            "id": 0,
+                            "lat": 59.917792,
+                            "lng": 10.754015
+                        },
+                        {
+                            "name": "Venedig Pizzeria Restaurang & Café",
+                            "category": "food",
+                            "id": 1,
+                            "lat": 56.045759,
+                            "lng": 12.697704
+                        },
+                        {
+                            "name": "Morten Volunteer",
+                            "category": "volunteer",
+                            "id": 2,
+                            "lat": 56.045759,
+                            "lng": 12.697704
+                        }
+                    ]
+                }
+            ]
+        };
+    }
+
   render() {
+      const { items } = this.state;
+      const test = items[0].items;
+
+      let pins = [];
+
+      for(let i =0; i < 10; i++) {
+          if(test[i]){
+              pins.push(
+                  <Pagination.Item key={i}>
+                  </Pagination.Item>
+              )
+          }
+      }
+
+      function setDetails() {
+          //process data and open up new screen
+      }
+
     return (
-      <Map
-        google={this.props.google}
-        zoom={14}
-        style={mapStyles}
-        initialCenter={{
-         lat: -1.2884,
-         lng: 36.8233
-        }}
-      />
+        <div>
+            <div id="pins-wrapper">
+                <h3>Filter pins</h3>
+                <Pagination className="map-pin">{ pins }</Pagination>
+            </div>
+            <Map
+                className = "map-container"
+                google={this.props.google}
+                zoom={12}
+                style={mapStyles}
+                initialCenter={{
+                    lat: -1.2884,
+                    lng: 36.8233
+                }}
+                googleMapURL={`https://maps.googleapis.com/maps/api/js?v=3.exp&libraries=geometry,drawing,places`}
+
+                {...test.map((pin) => (
```

Comment:
@tombjarne  BTW, I don't see the markers locally, may be because the API key is missing ? Should I try with a development key ?

Reply:
@santiagoarriaga me and @mahavir9008 will have a look into that this weekend, as there seems to be some problem with the access origin when accessing the map. Also I will have to change some things with the markers still.